cocotb/share/lib/vhpi/VhpiCbHdl.cpp

```diff
@@ -45,9 +45,12 @@ VhpiArrayObjHdl::~VhpiArrayObjHdl()
 
 VhpiObjHdl::~VhpiObjHdl()
 {
-    LOG_DEBUG("Releasing VhpiObjHdl handle at %p\n", (void *)get_handle<vhpiHandleT>());
-    if (vhpi_release_handle(get_handle<vhpiHandleT>()))
-        check_vhpi_error();
+    /* Don't release handles for pseudo-regions, as they use the handle of the containing region */
```

Comment:
Is this vhpi behavior or or own behavior?

Reply:
The pseudo-regions are an invention of cocotb for convenience.
I added a wiki page with explanation rather than detailing it here: https://github.com/cocotb/cocotb/wiki/cocotb-Internals